app/models/embassies_index.json

```diff
@@ -0,0 +1,3129 @@
+{
+  "content_id": "8e2cb457-81c1-484e-baa5-d714538c7c66",
+  "content": {
+    "title": "Find a British embassy, High Commission or Consulate",
+    "links": {
+      "parent": [
+
+      ]
+    },
+    "details": {
+      "lowercase_title": "Find a British embassy, high commission or consulate",
+      "world_locations": [
+        {
+          "name": "Afghanistan",
+          "has_consular_services": "remote",
+          "consular_services_organisations": [
+            {
+              "locality": "Dafna Area, Onaiza Zone 66, Al Shabab Street",
+              "name": "British Embassy Kabul",
+              "path": "/world/organisations/british-embassy-kabul"
+            }
+          ],
+          "remote_consular_services_organisation": {
+            "name": "British Embassy Kabul",
+            "country": "Qatar",
+            "path": "/world/organisations/british-embassy-kabul"
+          }
+        },
+        {
+          "name": "Albania",
+          "has_consular_services": "local",
+          "consular_services_organisations": [
+            {
+              "locality": "Tirana",
+              "name": "British Embassy Tirana",
+              "path": "/world/organisations/british-embassy-tirana"
+            }
+          ]
+        },
+        {
+          "name": "Algeria",
+          "has_consular_services": "local",
+          "consular_services_organisations": [
+            {
+              "locality": "Algiers",
+              "name": "British Embassy Algiers",
+              "path": "/world/organisations/british-embassy-algiers"
+            }
+          ]
+        },
+        {
+          "name": "American Samoa",
+          "has_consular_services": "remote",
```

Comment:
I think we're going to have to think about this quite carefully. I totally see what you were talking about at stand up. I'd like to see if there's a way that we can model this within World Locations. I realise that's a bit of a pain with places like American Samoa that don't have their own page and exist only as a content item, but I'd like to see if we can work out a way to do this without having to hardcode a field for each of these.

Reply:
Thanks @JonathanHallam! The discussion on the proposed schema was handled in https://github.com/alphagov/publishing-api/pull/2354, so I'll resolve this comment now.